deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -89,9 +81,7 @@ spec:
                 - image
                 type: object
               serviceAccount:
-                description: ServiceAccount refers to the kubernetes serviceaccount
-                  which is used for resource control. Default serviceaccount will
-                  be set if it is empty
+                description: ServiceAccount refers to the kubernetes serviceaccount which is used for resource control. Default serviceaccount will be set if it is empty
```

Comment:
we discussed this behaviour before in https://github.com/shipwright-io/build/pull/670 , can you take a look?

In a nutshell, in some systems when running the `controller-gen` we get line breaks ( _which is what most of us were seeing_ ) , however in others the line breaks are not happening.

Reply:
this PR kind of rollbacks that

Reply:
Weird, I have no idea why that would happen. I assumed it was some behavior change between v0.4.1 and v0.5.0. https://github.com/kubernetes-sigs/controller-tools/issues/514 indicates that report was traced to kustomize, but that wouldn't explain our behavior.

Honestly I think not line-wrapping seems better, who knows how those strings will be rendered by other tools, we shouldn't be making stylistic choices about how CRD YAML gets formatted, so long as the tool is consistent. Now if only the tool was consistent... 😒

Reply:
hmm wondering if we can externalize those CRDs generation into CI. For example: 

CI runs `make generate` -> on detected changes -> CI opens a PR with changes

I can merge this PR, but then someone will come with another PR that includes line breaks. 

Any ideas from your side on making it consistent?

Reply:
I think we should check on CI that codegen and CRD-gen are consistent with the Go source of truth, but not try to make anything generate and commit back changes. That way a human still has to run and check the output, but the generated code won't diverge from the source of truth in committed changes. wdyt?

Reply:
Re: making it consistent, I'd like to see if whatever issue you all were seeing was somehow fixed in v0.5.0, and the linebreak changes don't come back. If they do, we should be able to get more signal about where they're coming from.

Reply:
They were not fixed with `v0.5.0` for me. I will merge this PR, as the changes include a removal of some fields we don´t longer have and therefore not needed, as mentioned in https://github.com/shipwright-io/build/pull/786#discussion_r640015103 

Based on the current state ( _some users triggering `make generate-crds` that lead to new lines vs single lines_ ) we should allow only PRs with changes on the stylistic of the files, as long as this is done because of API changes. Any other PR that just wanna change the CRDs to a particular stylistic choice should be avoided ( _in my opinion_ ).

Reply:
based on my above comment, would it make sense to keep the changes in this PR only for `deploy/crds/shipwright.io_buildruns.yaml `, as this one includes updates on the API representation, besides the new lines. The rest of the files under `deploy/` are not required here, as they do not represent any change to the API representation, but rather changes on the file stylistic. Opinions?